### ISSUE TITLE
GvrCardboardHelpers.SetViewerProfile() (in Unity3d 5.6.1 with GVR 1.6) not working

### ISSUE THREAD

**Issue by peterept** [googlevr/gvr-unity-sdk#621]
I want to set the default viewer (as we are mailing out our own google cardboard headsets) and disable the default .

This was discussed for the previous SDK:
https://github.com/googlevr/gvr-unity-sdk/issues/158

However, in the new SDK either gvrViewer.cs or cardboard.cs are present any more.

I see in GoogleVR/Scripts/GvrCardboardHelpers.cs there is `gvr_set_default_viewer_profile`.

But it is not documented, nor clear when this call should be made (lets say it's in the Awake or Start) and whether that will hide the default cardboard dialog.

Calling GvrCardboardHelpers.SetViewerProfile() in Awake on IOS leads to this error at runtime:

```
DllNotFoundException: Unable to load DLL 'gvr': The specified module could not be found.
  at GvrCardboardHelpers.gvr_set_default_viewer_profile (IntPtr gvr_context, System.String viewer_profile_uri) [0x00000] in <filename unknown>:0 
```

Can we have a way to set the default like in the old SDK?

**Comment by peterept** [comment]
OK, found the issue: GvrSettings and GvrCardboardHelpers on ios should use [DllImport("__Internal")] not [DllImport("gvr")] 

I have created a pull request with the fix:

[Pull Request #622 ](https://github.com/googlevr/gvr-unity-sdk/pull/622)

**Some interesting factors that took me a while to figure this out:**

1. If you are using Unity VR profile "None" then calls like `GvrCardboardHelpers.SetViewerProfile()` will crash the app because the the VR native ptr `VRDevice.GetNativePtr()` (passed internally with calls such as gvr_set_default_viewer_profile) will be null for "None" profile. So don't call these cardboard specific methods when *not* in "cardboard" profile. 

Maybe an additional fix for safety is to check the current VR profile is "cardboard" before allowing these calls to proceed.

2. You must call `GvrCardboardHelpers.SetViewerProfile()`  in the `Awake()` method as `Start()` is too late. If you put it in Start, then you will still get the select profile dialog (unless the user has previous set it).

**Comment by fredsa** [comment]
Confirmed that the bug exists in the current 1.70 SDK.
This should be fixed in an upcoming release. Thanks PR #622.

**Comment by fredsa** [comment]
Fixed in v1.100.
